### ISSUE TITLE
SPARKLE Intel® Arc™ A750 ORC OC Edition stuck on 600/2000 mhz

### ISSUE THREAD

**Issue by Toetje585** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#821]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Windows / Linux

### Processor / Processor Number [Required]

 AMD Ryzen™7800X3D / AMD Ryzen™ 9 7945HX

### Graphic Card [Required]

SPARKLE Intel® Arc™ A750 ORC OC Edition

### GPU Driver Version [Required]

32.0.101.5762

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build [Required]

Windows 11 24H2

### Other Windows build

Windows 11 23H2

### Intel System Support Utility report

[intel.txt](https://github.com/user-attachments/files/16235425/intel.txt)

[B660DS3HAXDDR4.txt](https://github.com/user-attachments/files/16317568/B660DS3HAXDDR4.txt)

### Description and steps to reproduce [Required]

**Problem:** 

SPARKLE Intel® Arc™ A750 ORC OC Edition is always stuck on 600/2000 mhz on idle even without any displays connected in Windows and Also on Linux. Causing the fan to ramp up and down all the time.

**Reproduce:**

Install the SPARKLE Intel® Arc™ A750 ORC OC Edition, and use the latest drivers.

- Make sure ASPM enabled and is working
- Make sure the resolution is set to 1080p 60hz or lower.
- Obeserve the core/memory clock 600/2000
- Reboot the system and set the IGPU to primary in the bios.
- Shutdown the system.
- Disconnect all screens from the Intel® Arc™ A750 ORC OC Edition.
- Set the IGPU to primary and connect the HDMI/Displayport cable to the IGPU port.
- Boot the system, notice the ARC GPU is still running on 600/2000 without any screens connected.

Conclusion, even without any display connected the memory clock is not going down.

![1](https://github.com/user-attachments/assets/6211cfa8-f1e1-408d-9323-6f6903bd1e84)

Tested on 3 systems 2 (AMD) 1 (Intel) and also under Linux, e.g. my conclusion there is a firmware problem.
The above issue makes my fan ramp up and down all the time.

Best Regards,

### Device / Platform

BD790I and ROG STRIX B650E-E GAMING WIFI

**Comment by EstebanIntel** [comment]
Hi @Toetje585,

Can you clarify when you say, "Disconnect all cables from the Intel® Arc™ A750 ORC OC Edition." you include the power cables been disconnected?

**Comment by Toetje585** [comment]
@EstebanIntel Disconnect all screens from the card.

**Comment by Gabriela-Intel** [comment]
Hi @Toetje585. I tested out the issue and what I observed is that it was stuck at 600/2000 for about 2-3 minutes after booting but it would then drop to 0, remain there for 7-8 minutes, and then increase back up again for a few minutes before going back down to 0. And basically repeat itself again.

Is this similar to the behavior you're seeing or is it ALWAYS stuck at 600/2000?

**Comment by Toetje585** [comment]
Hi @Gabriela-Intel 

Thanks for testing this out.

I asume this test you did is also in a hybrid setup because you can see the gpu going of (0). I have the same behaivor here its. It always stuck at the highest state its either off or 600/2000 however because the gpu is setting the memory at 2000 and staying there for obiously no good reason the tempature reaches 50c fast and so the fan starts to ramp up and down. Can we somewhat agree this is strange behaivor specially because there no screens connected?

Does the card even have lower memory status at all?

**Comment by Gabriela-Intel** [comment]
Yes, definitely appears to be strange behavior. I filed a bug sighting so I can get more insight on this and perhaps get it fixed. Bug id is 14023014019 for your reference. Please stick around as it could take some time until we get more information. I'll post here again when I have an update!

**Comment by Gabriela-Intel** [comment]
Hey there @Toetje585. We have word back from the graphics debug team. Basically, this is expected behavior from the OS. What we're seeing is a momentary wake request being sent, which could be caused by Windows Update checking for a driver update for Arc. Every adapter has different wake timings so that's why the behavior might appear different than an Nvidia dgpu, for example.

You'll notice that there isn't a constant ramping up or down if you remove internet connection.

**Comment by Toetje585** [comment]
@Gabriela-Intel Doubt it, for my deployments Windows Update does not incude driver updates. And even if a wake request can happen the GPU memory is directly on highest memory state, this card is not capable of clocking down the memory and so heating up fast. The same behaivor is on Linux.  There is no need for the memory to run at highest powerstates if there no displays connected. The card is ramping up and down regardless of internet connection.

**Comment by Toetje585** [comment]
@Gabriela-Intel How are we going forward from here?

**Comment by Gabriela-Intel** [comment]
Can you please share the firmware ver currently installed? You can find it in Arc Control under Settings > System Info > GPU Info > IFWI.

I did share your comment with debug, and they plan on following up on this to continue further investigation.

**Comment by Toetje585** [comment]
@Gabriela-Intel Here is a detailed firmware overview:

Device: FW Version: DG02_1.3257
OPROM DATA Version: 14 00 24 04 00 00 00 00
OPROM CODE Version: 14 00 31 04 00 00 00 00
Device: Fw Data Version: Major Version: 101, OEM Manufacturing Data Version: 1, Major VCN: 1

Used: https://github.com/intel/igsc/

**Comment by Toetje585** [comment]
This is strange, the OPROM Code should be the same as OPROM DATA?

It seems like the DATA firmware is from Dec 12th, 2022, driver 101.3962 and the OPROM Code is from the latest driver. 101.5768.

**Comment by Toetje585** [comment]
@Gabriela-Intel I noticed something else, the card is not able to clean boot after a shutdown. For some reason sometimes the card is not detected in windows/linux if i start the computer again the **next morning.** However another powercycle brings it back. I tested this on two systems.

This is quite odd, also on lots of arc cards I see that OPROM DATA and OPROM CODE are on the same version, this seems not to be the case for this card.

**Comment by Toetje585** [comment]
@Gabriela-Intel Any update on this matter as this renders he card quite unusable?

**Comment by Gabriela-Intel** [comment]
Sorry for the delay. I've been reaching out to the team and haven't heard any updates on your issue. I'll follow up with them again and will post here once I have something to share.

**Comment by Toetje585** [comment]
@Gabriela-Intel ?

**Comment by Zack-Intel** [comment]
Hi @Toetje585 

Debug is still working on investigating this issue and analyzing their results to determine next steps. We'll keep you posted once there's more concrete information to share.

**Comment by Mickeythefallen** [comment]
Intel Arc a750 Sparkle ORC here.
Fans ramps up and stops in seconds non-stop. Nothing works to stop this (arc app, msi afterburner, evga precision x1) 
Months after months nothing fix this. This bug is killing me and my gpu. Please find a solution.

**Comment by Mickeythefallen** [comment]
Intel Arc a750 Sparkle ORC here.
Fans ramps up and stops in seconds non-stop. Nothing works (arc app, msi afterburner, evga precision x1 etc…) 
Please find a solution, this bug killing me and my gpu soon.

**Comment by Mickeythefallen** [comment]
Intel Arc a750 Sparkle ORC here.
Fans ramps up and stops in seconds non-stop. Nothing works (arc app, msi afterburner, evga precision x1 etc…) 
Please find a solution, this is killing me and my gpu.

**Comment by Toetje585** [comment]
> Intel Arc a750 Sparkle ORC here too. Fans ramps up and stops in seconds non-stop. Nothing works (arc app, msi afterburner, evga precision x1 etc…) GPU/Memory Clocks 600/2000 Mhz. So do we know if the source of the problem is software or hardware?

As it happens under linux, it looks like a firmware update.

**Comment by Mickeythefallen** [comment]
I've been researching this on the internet for a couple of weeks now, and there are a lot of users on Reddit and Intel forums who are experiencing this. Not only Sparkle products, but also acers, reference intel models.
What's strange is why this happens in some but not all arcs. I can't find anything to explain it.

**Comment by Toetje585** [comment]
> I've been researching this on the internet for a couple of weeks now, and there are a lot of users on Reddit and Intel forums who are experiencing this. Not only Sparkle products, but also acers, reference intel models. What's strange is why this happens in some but not all arcs. I can't find anything to explain it.

Looking at the info i have, it's all due the card staying at the highest performance state. It's known that with certain displays the memory clock is not clocking down however if i test my sparkle even without any monitor connected it's stays at the highest state.
I gues it's the same for you.

**Comment by Mickeythefallen** [comment]
After all these months, there is no explanation of the cause of my problem, let alone a solution.

**Comment by Mickeythefallen** [comment]
> > After all these months, there is no explanation of the cause of my problem, let alone a solution.
> 
> If you can confirm memory clock is at 2000 this is the cause. for the explanation we need to wait for @Zack-Intel Why this is happening for some vendors is most likely vendor related changes at firmware level.

yes man GPU/Memory Clocks 600/2000 Mhz. and the gpu has been in its box for 4 months because the gpu fans are driving me crazy.

**Comment by mujeeb-intel** [comment]
Hey @Toetje585 , I appreciate your patience on this matter.  Our team is working on this issue with Sparkle. 

I'll be sure to come back here and let you know if there is any new activity. We have this issue under our radar.

**Comment by dmarkey** [comment]
Problem on A310 ECO here.. ubuntu 24.04.. Fan very erratic as well power usage at the wall.

**Comment by Mickeythefallen** [comment]
Yes, friends...
After 1 year of not finding a solution to this problem, I don't think Intel will find a solution after this. The technical services are not willing to give a new product. I will try to sell the gpu myself at a loss and will never deal with intel again. Good luck to you.

**Comment by dmarkey** [comment]
Yeah, mine is going back to Amazon today.

**Comment by Toetje585** [comment]
@Mickeythefallen @dmarkey For Windows you can enable custom fan curve to workaround the annoying fan until Intel has a fix for us. For Linux users there is no fix as there is no way to control the fan. We need to wait for @mujeeb-intel to get back to us if a fix is possible. Regardless it's sad it takes a full year to get this to the right people but they seem to be on contact with Sparkle so it should be good now.

**Comment by Mickeythefallen** [comment]
> @Mickeythefallen @dmarkey For Windows you can enable custom fan curve to workaround the annoying fan until Intel has a fix for us. For Linux users there is no fix as there is no way to control the fan. We need to wait for @mujeeb-intel to get back to us if a fix is possible. Regardless it's sad it takes a full year to get this to the right people but they seem to be on contact with Sparkle so it should be good now.

No, strangely enough you can't do this in Windows either. Neither the fan curve nor the fixed fan speed works and the GPU/Vram speeds can never be changed. I tried everything in Win 10 and 11.

**Comment by Mickeythefallen** [comment]
Just for information. The only thing that stopped this strange fan behavior was to reduce the first 3 points in the fan curve settings to completely zero fan speed. In this case, the VRAM and GPU temperatures, which are already running at extremely high speeds, reach 75 degrees Celcius even when the card is idle.

**Comment by Toetje585** [comment]
> > @Mickeythefallen @dmarkey For Windows you can enable custom fan curve to workaround the annoying fan until Intel has a fix for us. For Linux users there is no fix as there is no way to control the fan. We need to wait for @mujeeb-intel to get back to us if a fix is possible. Regardless it's sad it takes a full year to get this to the right people but they seem to be on contact with Sparkle so it should be good now.
> 
> No, strangely enough you can't do this in Windows either. Neither the fan curve nor the fixed fan speed works and the GPU/Vram speeds can never be changed. I tried everything in Win 10 and 11.

No it won't change the vram speed but it will stop the fan turning of and on all the time.

**Comment by Gabriela-Intel** [comment]
Sorry for the lack of updates. I completely understand the frustration, it's taking a lot longer than we'd like to see some progress. What I can pass along is that the fan behavior for this card is not entirely controlled by our driver so we are working with Sparkle to investigate this.

Again, apologies for the lack of updates and thank you for sticking with us as we try to look into this.

**Comment by Toetje585** [comment]
> Sorry for the lack of updates. I completely understand the frustration, it's taking a lot longer than we'd like to see some progress. What I can pass along is that the fan behavior for this card is not entirely controlled by our driver so we are working with Sparkle to investigate this.
> 
> Again, apologies for the lack of updates and thank you for sticking with us as we try to look into this.

Thank you for the update, and I completely understand the frustration with the delays.

I just wanted to clarify that the issue you're seeing isn't related to the fan controller or its settings. After further investigation, it appears that the high power consumption of the card while idle is the main cause of the fans running at higher speeds. The card is drawing more power than expected in idle mode, which is triggering the fan to spin in an attempt to keep temperatures down.

As it's more about how the card's power management is handling idle states eg memory and core speed.

Let me know if you'd like more details or if there's anything else I can assist with!

**Comment by Gabriela-Intel** [comment]
Thanks for your understanding. Super appreciate it. :)

And yes I should have been clearer! I can confirm that the team is indeed looking into the high power consumption resulting in the card running hotter.

**Comment by Toetje585** [comment]
> Thanks for your understanding. Super appreciate it. :)
> 
> And yes I should have been clearer! I can confirm that the team is indeed looking into the high power consumption resulting in the card running hotter.

No new news for me ^^?

**Comment by Gabriela-Intel** [comment]
Hi. Sorry, no major updates I'm able to share at the moment. All I have is that we are going to collaborate with Sparkle to look into the issue.

**Comment by Gabriela-Intel** [comment]
@Toetje585  I totally get it. The issue has been ongoing since July. I'd ideally like to have seen this fixed by now but that isn't the case for this issue.
 
We try our best to be to be transparent and communicate but sometimes we are limited to what we can share, or there really aren't major updates that are worth sharing to the thread. All I can currently share is what was in the previous updates.
 
The debug and development team is constantly working on issues with varying priorities so sometimes it takes longer to see updates on some issues but that doesn't mean the issue won't get fixed or updated.
 
And as usual, we cannot promise an ETA of a fix. Trust me, I want to help move this along but our hands are tied.

**Comment by Mickeythefallen** [comment]
> This issue has been ongoing, and it’s completely fair to question whether it’s even fixable at this point. Given the amount of time that has passed, a clear resolution or at least a concrete update is long overdue.
> 
> If the issue is indeed complex and cannot be resolved through driver updates or firmware patches, Intel and SPARKLE should explore alternative solutions. This might include hardware swaps, offering replacement units, or collaborating on a tailored fix for affected users.
> 
> It’s crucial that both Intel and SPARKLE prioritize transparency about the root cause and the steps being taken. A lack of communication only adds to the frustration, and users deserve a timeline or at least an acknowledgment of alternative solutions being explored.
> 
> Please provide an update on whether this issue is actively being worked on and if users can expect a resolution soon. If not, alternative solutions should be on the table.

What happen after lastest driver update man? Is that solved?

**Comment by SavageWaffle** [comment]
I take it this is still ongoing with fix TBD

**Comment by Felipe-Intel** [comment]
@SavageWaffle This is still under investigation
@Gabriela-Intel please check.

**Comment by Toetje585** [comment]
> @SavageWaffle This is still under investigation
> @Gabriela-Intel please check. 

Please check if this is getting checked.

**Comment by Gabriela-Intel** [comment]
This is still open. I’ll share something here when we have any new updates.

**Comment by Toetje585** [comment]
This is still open, the card is 100% not usable due the constant fan ramping.

Can we get this resolved or do we need to work something out with a legal advisor?

**Comment by Gabriela-Intel** [comment]
@Toetje585 I would again like to thank you for your patience, and for checking in for updates. Although we do not post here regularly, our team continues to monitor this channel, so thank you for your updates and questions. The Intel development team is aware of the reported issue, however our team does not yet have a timeline for a potential fix to be implemented.

For your situation, if you are unable to wait for a resolution, Sparkle is the vendor and as such you may want to consider seeking a return through them, especially if your GPU is not performing according to your expectations.

**Comment by Toetje585** [comment]
@Gabriela-Intel Sparkle never responded to any of my emails, However Intel is responsible for the firmware and providing them to Sparkle.  If you reproduced the issue and claim to be in contact with them please provide meaningful updates.

The issue has been open now for 240 days.... and the card is idling at 45/50W due the memory not clocking down, it should be possible for Intel GPU's if you enable ASPM....

**Comment by Toetje585** [comment]
![Image](https://github.com/user-attachments/assets/f80f2b94-da17-400d-8177-622794bc6efd)
![Image](https://github.com/user-attachments/assets/0676ee27-ce19-41d1-b80a-267ea7126dde)
![Image](https://github.com/user-attachments/assets/d665da7b-18cd-4997-ba30-81f48c4e1c0a)

Here are some ASPM screenshots.

**Comment by Gabriela-Intel** [comment]
Thanks again for reporting this issue and thank you for your patience.

Our priority is to target the most popular games and apps to focus our efforts on providing a high quality, stable experience for the broadest set of users. We will continue to improve our software performance and compatibility throughout this year and beyond.

While we can’t accommodate your request at this time, please watch this [article ](https://www.intel.com/content/www/us/en/support/articles/000099450.html)on our website for any possible changes to this situation.
